app/src/main/res/layout/login_fragment.xml

```diff
@@ -3,124 +3,114 @@
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size30"
-        android:ems="10"
-        android:textColorHint="@color/black"
-        android:textColor="@color/black"
-        android:hint="@string/phone"
-
-        android:inputType="phone"
-        app:layout_constraintEnd_toEndOf="@+id/btnGoogle"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tvOr" />
-
-    <Button
-        android:id="@+id/btnLogin"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size30"
-        android:text="@string/login"
+        android:layout_marginTop="@dimen/margin99"
+        android:layout_marginLeft="@dimen/margin16"
+        android:layout_marginRight="@dimen/margin16"
+        app:boxBackgroundColor="@color/azure"
+        app:errorEnabled="true"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/editPhone" />
-
-    <TextView
-        android:id="@+id/tvSSO"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size20"
-        android:text="@string/sso"
-        android:textColor="#0B8CF3"
 
-        android:textSize="@dimen/text20"
-        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/btnLogin" />
+        app:layout_constraintTop_toBottomOf="@+id/tvLogin">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/phone"
+            android:inputType="number"
+            android:textColor="@color/black"
+            android:textColorHint="@color/black" />
+    </com.google.android.material.textfield.TextInputLayout>
 
     <TextView
-        android:id="@+id/tvForgot"
+        android:id="@+id/tvHelp"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size20"
-        android:text="@string/forgot"
-        android:textColor="#0B8CF3"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.146"
+        android:layout_marginTop="@dimen/margin16"
+        android:text="@string/help"
+        android:textColor="@color/azure"
+        android:textSize="@dimen/text12"
+        android:layout_marginLeft="@dimen/margin288"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tvSSO"
-        app:layout_constraintVertical_bias="0.268" />
+        app:layout_constraintTop_toBottomOf="@+id/editPhone" />
 
     <TextView
```

Comment:
@raghav32 according to the design, info text and proceed_securely button are supposed to be aligned on the bottom of the screen, but you are showing below the need_help text. Please fix this